app/jobs/report_export_uploader_job.rb

```diff
@@ -26,7 +26,7 @@ def save_tempfile(export)
   end
 
   def upload_csv_to_s3(file:, filename:)
-    Export::S3Uploader.new(file: file, filename: filename, use_public_bucket: false).upload
+    Export::S3Uploader.new(file: file, filename: filename).upload
```

Comment:
Here again I would like to see more in the commit message, I appreciate this may be a taste thing.